cmd/config/internal/commands/fmt.go

```diff
@@ -59,13 +67,13 @@ func (r *FmtRunner) preRunE(c *cobra.Command, args []string) error {
 }
 
 func (r *FmtRunner) runE(c *cobra.Command, args []string) error {
-	f := []kio.Filter{filters.FormatFilter{
+	r.filters = []kio.Filter{filters.FormatFilter{
```

Comment:
Why are we creating the filters here? Seems like we can just create them on the fly in `executeCmd`. Filters might also have state, so it is not clear that it is always safe to reuse them.

Reply:
@mortent Done.